source/Components/AvalonDock/Controls/LayoutFloatingWindowControl.cs

```diff
@@ -248,7 +250,9 @@ protected virtual IntPtr FilterMessage(IntPtr hwnd, int msg, IntPtr wParam, IntP
 							handled = true;
 						}
 					}
-					break;
+                    UpdateWindowsSizeBasedOnMinSize();
```

Comment:
This is the first location where the style was loaded and the grid and border size could be retreived